packages/react-form-state/src/components/List.tsx

```diff
@@ -10,10 +10,21 @@ interface Props<Fields> {
   children(fields: FieldDescriptors<Fields>, index: number): React.ReactNode;
 }
 
-export default class List<Fields> extends React.PureComponent<
+export default class List<Fields> extends React.Component<
   Props<Fields>,
   never
 > {
+  shouldComponentUpdate(nextProps) {
```

Comment:
is it possible for `initialValue` to change but value & error to stay the same?
mostly thinking if something is displayed based on the `initialValue`

Reply:
it *is* possible but it's a pretty weird edgecase